### PR TITLE
feat: Docker container-init.d (go-ipfs 0.13)

### PR DESCRIPTION
**Pull request by guseggert**

### CODE REVIEW

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Comment by guseggert** [comment]
This shouldn't be merged until this feature is released in go-ipfs v0.13

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Comment by johnnymatthews** [comment]
> This shouldn't be merged until this feature is released in go-ipfs v0.13

Converted to draft in the meantime.

File: docs/how-to/run-ipfs-inside-docker.md [hunk 1]
```diff
@@ -71,6 +71,33 @@ When starting a container running ipfs for the first time with an empty data dir
...
+docker run -d --name ipfs \
+  -e PRIVATE_PEER_ID=... \
+  -e PRIVATE_PEER_IP_ADDR=... \
+  -v ./001-test.sh:/container-init.d/001-test.sh \
+  -p 4001:4001 \
+  -p 127.0.0.1:8080:8080 \
+  -p 127.0.0.1:5001:5001 \
+  ipfs/go-ipfs
+```
+
```
Comment (lidel) [inline docs/how-to/run-ipfs-inside-docker.md:99]: Not sure if this is the right place, but I'd like to point people at examples repo created by @thattommyhall 

```suggestion

::: tip Use in custom images

See `gateway` example at https://github.com/ipfs-shipyard/go-ipfs-docker-examples

:::
```

**Review by lidel — APPROVED** [review]

File: docs/how-to/run-ipfs-inside-docker.md (hunk 1 above)
Comment (johnnymatthews) [reply docs/how-to/run-ipfs-inside-docker.md:99]:
```suggestion
::: tip Use in custom images
See `gateway` example at[github.com/ipfs-shipyard/go-ipfs-docker-examples](https://github.com/ipfs-shipyard/go-ipfs-docker-examples)
:::
```

Sure, but let's format it like this.

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Pull request merged**